dotCMS/src/main/java/com/dotcms/rest/api/v1/secret/DeleteSecretForm.java

```diff
@@ -0,0 +1,41 @@
+package com.dotcms.rest.api.v1.secret;
+
+import com.dotcms.repackage.javax.validation.constraints.NotNull;
+import com.dotcms.rest.api.Validated;
+import java.util.Set;
+
+public class DeleteSecretForm extends Validated {
```

Comment:
we used to use a builder or constructor with json creator to make it immutable

Reply:
I would say if the 
```
checkValid();
```
Is not call on the constructor won't do the null checkings